src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/command/flow/FlowSegmentReport.java

```diff
@@ -89,16 +90,18 @@ private SpeakerResponse makeSuccessReply() {
                 .build();
     }
 
-    protected void decodeError(FlowErrorResponseBuilder errorResponse, OFErrorMsg error) {
+    protected static void decodeError(FlowErrorResponseBuilder errorResponse, OFErrorMsg error) {
```

Comment:
Could you please argue this?

Reply:
I support the idea to extract it to separate util class. 
or to avoid static keyword. 
My argument might be to extract util-related logic to separate util classes and not use static methods in classes which are mainly used as an instances.  But this is only my imho.